### PR TITLE
Taverly dungeon

### PR DESCRIPTION
**Pull request by alycii**
## What has been done?
- Added all spawns
- Added definitions for hellhounds, monk of zamorak, chaos dwarfs, and the Jailer
- Added dialogue for obtaining the Dusty Key
- Added support for Jail & Dusty Key
- Fixed two critical bugs with doors and walkable tiles

## Has your code been documented?
- Mostly

### CODE REVIEW

**Comment by DesecratedTree** [comment]
pulled to local, did not run into anything major, still a work in progress. approving

**Comment by skalfate** [comment]
One small typo error in taverley plugin file, line 44 "well" is "wll"

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/taverley/taverly_dungeon.plugin.kts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+on_npc_option(npc = Npcs.VELRAK_THE_EXPLORER, option = "talk-to") {
+    player.queue {
+        if (!player.inventory.contains(Items.DUSTY_KEY)) {
+            chatNpc("Thank you for rescuing me! It isn't very comfy in this", "cell!")
+            when (options("So... do you know anywhere good to explore?", "Do I get a reward?")) {
+                FIRST_OPTION -> {
+                    chatPlayer("So... do you know anywhere good to explore?")
+                    chatNpc(
+                        "Wll, this dungeon was quite good to explore ...until I",
```
Comment (skalfate) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/taverley/taverly_dungeon.plugin.kts:40]: typo Wll shoul be well

**Review by skalfate — CHANGES_REQUESTED** [review]
Looks very well done! Just a small typo away from perfect.

**Pull request merged**